next-env.d.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
This changes is generated on `npm run build`, because `useRouter` does not have to support both `pages/` and `app/`